packages/instrument-utils/src/storedInstrument/InstrumentStoreAsDefaultField/InstrumentStoreAsDefaultField.tsx

```diff
@@ -1,21 +1,30 @@
-import React, { FunctionComponent, memo, useMemo } from 'react';
+import React, { FunctionComponent, memo, useEffect, useMemo } from 'react';
 
 import { TranslatedString } from '@bigcommerce/checkout/locale';
 import { CheckboxFormField } from '@bigcommerce/checkout/ui';
+import { Formik } from 'formik';
 
 interface InstrumentStoreAsDefaultFieldProps {
     isAccountInstrument: boolean;
     disabled?: boolean;
+    formik?: Formik;
 }
 
 const InstrumentStoreAsDefaultField: FunctionComponent<InstrumentStoreAsDefaultFieldProps> = ({
     isAccountInstrument,
     disabled = false,
+    formik,
```

Comment:
Can you please try and use https://github.com/bigcommerce/checkout-js/blob/master/packages/payment-integration-api/src/contexts/payment-form-context/PaymentFormContext.tsx#L9 instead of accessing formik directly. This gives a facade to some pre-defined method in `PaymentFormService`.